linode/objbucket/resource.go

```diff
@@ -74,6 +74,7 @@ func readResource(
 	// Functionality requiring direct S3 API access
 	accessKey := d.Get("access_key").(string)
 	secretKey := d.Get("secret_key").(string)
+	endpoint := d.Get("endpoint").(string)
```

Comment:
We can resolve the endpoint from the bucket hostname here:

```suggestion
	endpoint := strings.TrimPrefix(bucket.Hostname, fmt.Sprintf("%s.", bucket.Label))
```